compiler/rustc_borrowck/src/region_infer/values.rs

```diff
@@ -158,9 +141,9 @@ impl<N: Idx> LivenessValues<N> {
 
     /// Adds all the elements in the given bit array into the given
     /// region. Returns whether any of them are newly added.
-    crate fn add_elements(&mut self, row: N, locations: &HybridBitSet<PointIndex>) -> bool {
+    crate fn add_elements(&mut self, row: N, locations: &IntervalSet<PointIndex>) {
```

Comment:
since `SparseIntervalMatrix::union_row` returns a bool like `SparseBitMatrix` did, did this signature need to change to not return a bool anymore ? 

If that's the case, "Returns whether any of them are newly added" in the doc comment above can be removed.